ci/azure-pipelines-dev.yml

```diff
@@ -251,6 +254,23 @@ stages:
               pytest -v --no-sqlalchemy --docs-tests -m integration -k "RUNME" tests/integration/test_script_runner.py
             displayName: 'Run Flagged Tests'
 
+      - job: runme_script_runner_tests_postgres
+
+        steps:
+          - task: UsePythonVersion@0
+            inputs:
+              versionSpec: '3.7'
+            displayName: 'Use Python 3.7'
+
+          - script: |
+              pip install --constraint constraints-dev.txt ".[test, postgresql]"
+            displayName: 'Install GX and required dependencies (i.e. not sqlalchemy)'
+
+          # tests containing the name "RUNME_POSTGRES" are run
+          - script: |
+              pytest -v --postgresql --docs-tests -m integration -k "RUNME_POSTGRES" tests/integration/test_script_runner.py
```

Comment:
Wanting to check : is `RUNME_POSTGRES` the name we are keeping?

Reply:
Good question - the "RUNME" prefixes will get removed once docs_integration tests are running again. This is just a workaround until then!